php/libraries/CouchDB.class.inc

```diff
@@ -725,7 +740,7 @@ class CouchDB
      *
      * @return string The string that was returned by CouchDB
      */
-    function commitBulkTransaction()
+    function commitBulkTransaction(): string
```

Comment:
the line right below this is "return false"

Reply:
Thanks! I generated this with the PHPCBF tool and it just pulls the return type from the PHPDOC.